src/example_node_wallet.ts

```diff
@@ -0,0 +1,27 @@
+import { Keypair, TransactionBuilder, hash } from '.'
```

Comment:
can we put these all in a subfolder to group them together in the repo? I don't like having an example in the top-level

Reply:
Great idea. Moved to the `contract_client` folder.